theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/shared.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * This 4-tuple defines the start control point x1,y1 and the end control point x2,y2
  * of a cubic bezier curve. It is assumed that the start of the curve is fixed at 0,0
- * and the end is fixed at 1,1. x1 must always be >= 0 and x2 must always be <= 1.
+ * and the end is fixed at 1,1. x1 must always be \>= 0 and x2 must always be \<= 1.
```

Comment:
Nit
```suggestion
 * and the end is fixed at 1,1. X values must be constrained to `x1 >= 0` and `x2 <= 1`.
```

Reply:
Thanks, this was actually incomplete. They should both be in the 0, 1 interval